extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-date-picker"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {
+  html_format: AMP
+  spec_name: "amp-date-picker[type=single][mode=static]"
```

Comment:
`spec_name` should come after `tag_name`.

Reply:
Is this requirement documented somewhere or is there something intuitive that I'm missing?

Reply:
It's not documented documented. We (or at least I) try to keep the fields in the order they are in the [proto](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L385). It makes it easier to read various protoasciis after one has looked at a lot of them.

Reply:
ah that makes sense 👍